ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py

```diff
@@ -51,17 +51,43 @@ def test_tensorflow_linear(dtype_and_x, as_variable, num_positional_args, native
     )
 
 
+# relu
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float")),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.relu"),
+)
+def test_tensorflow_relu(
+        dtype_and_x, as_variable, 
+        num_positional_args, native_array):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="tensorflow",
+        fn_tree="keras.activations.relu",

```

Comment:
you should change this to `nn.relu`

Reply:
you mean i have to change  fn_tree="keras.activations.relu" to fn_tree="tf.nn.relu" ?

Reply:
> Hi, Just a few changes requested, can you also remove changes from other functions.
> 
> thanks

I was getting flake8 format errors from other functions also.That's why I edited the format of other functions as well.Should I remove it?

Reply:
> you mean i have to change fn_tree="keras.activations.relu" to fn_tree="tf.nn.relu" ?
you should do `fn_tree="nn.relu"`

Reply:
> I was getting flake8 format errors from other functions also.That's why I edited the format of other functions as well.Should I remove it?

Yes its better to do them separately and not in a PR of another frontend function

Reply:
ok